test/helpers/utils/rpc.js

```diff
@@ -5,7 +5,7 @@ const pify = require("pify");
  * @param {Object} provider Ganache provider
  * @returns {Promise} Response object
  */
-const send = (provider) => (method = "", ...params) => {
+const generateSend = (provider) => (method = "", ...params) => {
```

Comment:
I understand the need for backwards compatibility while we upgrade our tests, but `generateSend` and `send` are nearly identical. Can you DRY this up a bit?

Reply:
Sure, let me check with @nicholasjpaterno on some of his finer points.

Reply:
It turns out we can export a `send` function when we `initializeTestProvider`. Keeping `generateSend`.